packages/paste-core/components/typography/rollup.config.js

```diff
@@ -2,20 +2,19 @@ import typescript from 'rollup-plugin-typescript2';
 import babel from 'rollup-plugin-babel';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
-import {
-  terser
-} from 'rollup-plugin-terser';
+import {terser} from 'rollup-plugin-terser';
```

Comment:
is it weird that prettier suddenly thought this wasn't OK?

Reply:
It is lol.  I don't really question prettier.